src/ru/desu/src/eu/kanade/tachiyomi/extension/ru/desu/Desu.kt

```diff
@@ -363,13 +365,29 @@ class Desu : ConfigurableSource, HttpSource() {
                 true
             }
         }
+        val domainDesuPref = EditTextPreference(screen.context).apply {
```

Comment:
consider implementing [this](https://github.com/keiyoushi/extensions-source/blob/68e73a297f229a7a7b2dc675f1aacc0e17e061f5/src/ru/rumix/src/eu/kanade/tachiyomi/extension/ru/rumix/RuMIX.kt#L51-L77) so extension update changing the domain actually changes it for people.

idk why this comment didn't get published in review last time, sorry

Reply:
Maybe later. For now, just merge

Reply:
And I don't understand how to use it not in a multi source. The current setup has been working for many years in many extensions.